frontend/src/ui/Blocks/VideoList.tsx

```diff
@@ -116,6 +144,34 @@ export const VideoListBlock: React.FC<VideoListBlockProps> = ({
 
     const eventsNotEmpty = items.length > 0;
 
+    // I don't like this but i18next makes your life kinda hard when it comes to passing two counts
+    // and translating all possible pairs of each count being either 0, 1 or greater than 1,
+    // also considering the fact that a "1" should be shown as "one" or "One".
+    // While we can implement lower/upper case formatting in i18next, this only works for strings
+    // and will fail if we try to pass a count.
+    const hiddenItemsTranslation = () => {
+        if (missingItems === 1 && unauthorizedItems >= 1) {
+            return t("videolist-block.hidden-items.one-to-n", { count: unauthorizedItems });
+        }
+        if (missingItems > 1 && unauthorizedItems === 1) {
+            return t("videolist-block.hidden-items.n-to-one", { count: missingItems });
+        }
+        if (missingItems > 1 && unauthorizedItems > 1) {
+            return t("videolist-block.hidden-items.n-to-m", {
+                missingCount: missingItems,
+                unauthorizedCount: unauthorizedItems,
+            });
+        }
+        if (missingItems >= 1 && unauthorizedItems === 0) {
+            return t("videolist-block.hidden-items.missing", { count: missingItems });
+        }
+        if (missingItems === 0 && unauthorizedItems >= 1) {
+            return t("videolist-block.hidden-items.unauthorized", { count: unauthorizedItems });
+        }
+        return;
+    };
```

Comment:
It is indeed quite some manual logic written here. To be honest, I would probably reduce all of this by simply having one info-box for each of the two kinds of hidden items. So in some cases, two boxes will appear. But that's fine I think. Also considering that both of these cases are not too common. But yes, that lets us shrink this logic and the number of translations considerably.